src/ophys_etl/decrosstalk/decrosstalk_schema.py

```diff
@@ -0,0 +1,121 @@
+import marshmallow.exceptions
+import argschema
+
+
+class RoiSchema(argschema.ArgSchema):
```

Comment:
I think you're using this just as nested, rather than through a parser. It will work, but, `argschema.schemas.DefaultSchema`  is more appropriate.
Same for the next 2 schemas as well.
https://github.com/AllenInstitute/argschema/blob/afc2b9241176a587e543f71830f2968d7a552cb7/argschema/schemas.py#L5

Reply:
I've updated all of the "internal" schemas to inherit from `argschema.DefaultSchema`